.github/workflows/release.yaml

```diff
@@ -29,6 +29,7 @@ jobs:
 
       - name: configure git
         run: |
+          echo "GITHUB_ACTOR=${GITHUB_ACTOR}"
```

Comment:
**question (non-blocking):** is this debugging needed?

Reply:
Useful. We can remove it later.